### PR TITLE
Remove redundant imports and code. (Linting)

### PR DESCRIPTION
**Pull request by pmellati**
Brought to you by my atom IDE.

### CODE REVIEW

File: app/DockerFu.hs [hunk 1]
```diff
@@ -85,7 +80,7 @@ parse = info (helper <*> parseArgs)
               <> header "docker-fu: build magic for the awesome enterprise"
              )
 
-dockerTodo :: (Member Trace r)
```
Comment (jberthold) [inline app/DockerFu.hs]: This (part of this) file will change substantially in a forthcoming pull request.
Can we take it (and `docker-fu.hs`) out of the PR please?

In general I am a bit in doubt about clean-up only" patches

File: app/docker-fu.hs [hunk 2]
```diff
@@ -4,10 +4,6 @@
 
 module Main where
 
-import Options.Applicative
```
Comment (jberthold) [inline app/docker-fu.hs]: Same comment here: please do not edit this file yet.

**Review by jberthold — CHANGES_REQUESTED** [review]
Generally unsure about doing this just yet (although it is great Puria got a nice setup to put it together.
And, please do not change the main application, it is far from ready.

**Comment by pmellati** [comment]
@jberthold I've taken the two "docker fu" files out of the PR.

**Comment by jberthold** [comment]
OK, thanks for revisiting. LGTM now

**Review by jberthold — APPROVED** [review]

**Pull request merged**